InstallService/ConfigInstall.cs

```
@@ -16,16 +17,24 @@ public static class ConfigInstall
         public static bool ValidateAndWrite(string configFilePath, string oktaUrl,
             string clientId, string clientSecret, out string errorMessage)
         {
-            // validate
-            OktaMfaConfig config = new OktaMfaConfig(oktaUrl, clientId, clientSecret);
-            if (!IsValidConfig(config, out errorMessage))
+            try
             {
-                return false;
-            }
+                errorMessage = String.Empty;
 
-            // write to disk
-            config.Write(configFilePath);
-            return true;
+                // validate
+                OktaMfaConfig config = new OktaMfaConfig(oktaUrl, clientId, clientSecret);
```

I found I could not easily catch exceptions thrown across DLL boundaries - that's why I went with _out errorMessage_.  But maybe that's just with the ADFS plugin.

Have you tried throwing an exception in _OktaMfaConfig_ to confirm it doesn't silently fail?